### DISCUSSION TITLE
Block a tv-show to only search in a selected provider

### DISCUSSION THREAD

**Discussion by DAVIZINH0** [pymedusa/Medusa#8465] [Ideas & Feature requests]
**Is your feature request related to a problem? 
NO, this is not a problem. its only something very usefull

**Describe the solution you'd like**
The context is the next:
- I have 3 diferent providers
- I have a tv-show that i want to search ONLY in one of this providers, because in 2 of this providers apears in 4K, but ONLY in one of this apears in 4K HDR.

The solution for me will be:
- When i add the tv-show i mark the quality and the exception.
- By default all the providers are selected
- I can select only one of the providers ONLY for this tv-show

Thanks for your work, medusa is for me the best of this segment!!! thanks a lot!

**Comment by ciltocruz** [comment]
Great Idea!
+1

**Comment by p0psicles** [comment]
So this is just to save some api hits on the providers that are not selected?

**Comment by medariox** [comment]
Put that provider at the top of the providers list. That way it won't query the other providers if the wanted result is found.

**Comment by DAVIZINH0** [comment]
Sorry, maybe my english is so poor :-( and i dont understand your question.

Now a show allways search chapters in all the providers are selected in the order of priority you selected in the providers menu. this is ok.
This causes that in some cases, the tv-show download a lot of times because, some providers publish in low quality (not hdr for example), and some days after, the favorite provider publish in better quality (hdr for example)

But in a concret tv-shows i what to search only in one provider because i know that this provider is the best for this concret tv-show.

My idea is add a combo in the menu "advanced settings" on each tv-show that you can select ONLY ONE provider.
After that, this tv-shows in automatic searched task, ONLY search this tv-show in this provider.

thanks for reply!!

**Comment by p0psicles** [comment]
And if you use required words, and set to 'hdr'. Or use the release group. That way you get the same result right?

**Comment by DAVIZINH0** [comment]
the hdr was a example. the intention is mark a show to download only from one tracker because this tracker have best quality or best bitrate, etc....
maybe this was a stupid idea if only i see the benefict jejeje

**Comment by p0psicles** [comment]
Medusa should pick your desired quality. Independent of the provider. So Im challenging your request, to see if there aren't other ways, to get what you aim for.

**Comment by p0psicles** [comment]
You could even put a specific provider on a delay

**Comment by medariox** [comment]
As explain, there are already plenty of ways to handle the situation. Closing.

**Comment by p0psicles** [comment]
We don't want to close down the conversation though. But it's very easy to think of cool features. It's not that easy to implement them. Or when implemented have them make sense to new users.

**Comment by DAVIZINH0** [comment]
I still think that the actual features dont cover the exact case that i describe :-(
today i talk about this with a friends of telegram and apears other interesting example:
A lot of people use public trackers and private trackers in medusa
The private trackers are very "dangerous" because if medusa download a lot of chapters you can destroy your ratio. :-(
The most people dont use medusa with this private trackers and download some tv-shows manually.
The user say "I only want to search this tv-show in this private tracker, because the quality is perfect". BUT i dont want to search all my movies in this private tracker.

In this case it will be very usefull to block one tv-show to only one search provider (the private). maybe more tv-shows but onlyl a few, the most still searching in the public trackers.

For this "example", the users now have 2 medusas in dockers. one for public trackers and one for each private tracker. 
This isn´t optimal at all :-(

Other interesting idea if this one was posible will be to set the "postprocesor method" for each provider:
- Public trackers: move in postprocess
- private trackers: symlink in postprocess (for share a long time each download)

Sorry por a long reply, and a big big sorry for my poor english, I have a lot of ideas but i have dificult to expain :-)

regards!

EDIT: your work is excelent! and medusa is a excelent software. im very happy with all of your support and continuous updates!!! congrats for a big big work!!!

**Comment by moralia** [comment]
I think it would be a very interesting function. The same thing happens to me, with private trackers there is a "danger" of losing the ratio easily, and it would be a great improvement to add this option!

+1!

**Comment by Patxinho** [comment]
Great Idea!
+1

**Comment by SacalmM** [comment]
It's a great idea! Now, I'm using several docker containers with Medusa to be able to use different trackers for specific series, but it is an unnecessary waste of resources

**Comment by medariox** [comment]
@DAVIZINH0
What you describe can already be done by setting a delay for private trackers.

**Comment by Vctrsnts** [comment]
I have read @DAVIZINH0 explanation and I find it interesting because if you can select the language with which you want the series to search for you, it is also a good idea that you can choose a default tracker.

If it could be done, it would "save resources" for the series search, because you would already say that it would only look for you in a specific tracker and not in all those that you have activated.

**Comment by DAVIZINH0** [comment]
> 
> 
> @DAVIZINH0
> What you describe can already be done by setting a delay for private trackers.

Hello!
But a delay may works in some cases, but all the tv-shows make searches (with diferents delays) in all the providers, include in the private.
What i say, the tv-show that is market only in a private tracker, only search in this private tracker. never search in other trackers. An for the rest of series if i not marked the private tracker, never search in the private trackers.

I love this debates!!! :-)
